go/chat/server.go

```diff
@@ -471,10 +479,24 @@ func (h *Server) GetThreadNonblock(ctx context.Context, arg chat1.GetThreadNonbl
 			return
 		default:
 		}
-		h.Debug(ctx, "GetThreadNonblock: cached thread sent")
+		var pthread *string
+		if resThread != nil {
+			h.Debug(ctx, "GetThreadNonblock: sending cached response: %d messages", len(resThread.Messages))
+			var jsonPt []byte
+			var err error
+			pt := h.presentThreadView(*resThread)
+			if jsonPt, err = json.Marshal(pt); err != nil {
+				h.Debug(ctx, "GetThreadNonblock: failed to JSON cached response: %s", err)
+				return
```

Comment:
Yes, this is the "cached" fetch (which happens in parallel), so if any error comes out of it we just swallow it. The expectation is that this guy only returns an error if we weren't able to get the "full" result up.